integration/generic-tests/kexec_test.go

```diff
@@ -51,7 +51,7 @@ func TestMountKexec(t *testing.T) {
 // kexec_load syscall than file load.
 func TestMountKexecLoad(t *testing.T) {
 	// TODO: support arm
-	if vmtest.TestArch() != "amd64" {
+	if vmtest.TestArch() != "amd64" && vmtest.TestArch() != "arm64" {
```

Comment:
hey, just checking, did you consider
kexec_test_linux.go for file name, and
// +build arm64, amd64
in the file, so you could not even need lines 54-56?

Reply:
Done. 

I thought I wouldn't be able to make the test work. but now it also works on `arm64`, yea.

Reply:
OK.... build constraints does not work here... it seems that all our test Docker are amd ... Arm (and Arm64) are all cross compiled...

Arm tests are always built, and run. 

We don't support arm kexec yet... so it will just fail there. 

Gonna need those If explicit checks ...